charts/influxdb2/templates/statefulset.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1beta2
+kind: Deployment
```

Comment:
File name is statefulset, but, the kind is Deployment.

I think it makes sense for it to be a deployment rather than a statefulset.